central/buildbot.py

```diff
@@ -32,8 +32,8 @@ def make_build_request(repo, pr_id, job_id, baserev, headrev, patch, who,
         'builderNames': cfg.buildbot.pr_builders,
         'jobid': job_id,
         'baserev': baserev,
-        'patch_level': 1,
-        'patch_body': patch,
+        'patch_level': 0,
+        'patch_body': '',
```

Comment:
The ```patch_body``` should probably be ```None``` according to [the documentation](http://docs.buildbot.net/latest/developer/database.html?highlight=patch_body).

Ditto for the ```patch_level```?